README.md

```diff
@@ -352,6 +340,92 @@ Returns one of the following in order of preference:
 In order for this to be most useful, you must configure your controller
 as described in [Change Journaling](#change-journaling) above.
 
+### Testing
+
+If you use RSpec, you can test for journaling behaviors with the
+`journal_event(s)` and `journal_changes_to` matchers. First, make sure to
+require `journaled/rspec` in your spec setup (e.g. `spec/rails_helper.rb`):
+
+```ruby
+require 'journaled/rspec'
+```
+
+#### Checking for specific events
+
+The `journal_event` and `journal_events` matchers allow you to check for one or
+more matching event being journaled:
+
+```ruby
+expect { my_code }.to journal_event(name: 'foo')
+expect { my_code }.to journal_events({ name: 'foo', value: 1 }, { name: 'foo', value: 2 })
+```
+
+This will only perform exact matches on the specified fields (and will not match
+one way or the other against unspecified fields).
```

Comment:
> is it worth considering making the partial-match quality clearer by naming the matcher journal_event_matching or something?

I thought about it. I was thinking `journal_event_including`. It sort of begs the question of whether there is utility in an exact match option too -- I was struggling to find a case where we'd want one (and in every case we want to exclude git commit, the random UUID, and probably the created_at timestamp, so it would never be an exact match).

What about the alternative, of forcing the consumer to use RSpec argument matchers more explicitly:

```ruby
expect { ... }.to journal_event(hash_including(...)) # fuzzy match
```

vs.

```
expect { ... }.to journal_event(...) # exact match, excluding difficult fields
```

Reply:
(cc @jmileham ^^)

Reply:
I'm fine where you ended up, I agree that exact match is not highly valuable. I would be ok with an `_including` suffix on the name with no exact match variant available.

Reply:
Optionally add an _including suffix, or leave it as is.